modules/nw-create-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-create-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a new custom Ingress Controller. Because the default Ingress Controller might be updated during {product-title} upgrades, creating a custom Ingress Controller can be helpful when maintaining a configuration manually that persists across cluster upgrades.
+
+This example provides a minimal spec for a custom Ingress Controller. To further customize your custom Ingress Controller, see "Configuring the Ingress Controller".
```

Comment:
@melvinjoseph86 please specifically review the concepts presented here as this is accurate to last known info, please confirm.
@jeana-redhat just so you can follow along